### PR TITLE
Make install instructions more clear.

### PR DESCRIPTION
**Pull request by driesvints**
Make it more clear how to install this in a project which uses Composer. Apparently it's enough to have this package in your dependencies but the install instructions from the readme weren't clear on this.

https://symfony.com/doc/current/setup.html#checking-for-security-vulnerabilities

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ Integration
 The checker uses the Symfony Console component; so, you can easily integrate
 the checker into your own project:
 
+ * by requiring it with Composer: `composer require ensiolabs/security-checker --dev`: https://symfony.com/doc/current/setup.html#checking-for-security-vulnerabilities
```
Comment (fabpot) [inline README.md]: I would not recommend it as your project deps can interfere with the checker ones.

Comment (driesvints) [reply README.md]: But it's in Symfony's own docs? Does it need to be removed from those docs then if you don't recommend it?

Comment (fabpot) [reply README.md]: It needs to be removed everywhere. It's the default value, so there is no need to be explicit here.

Comment (driesvints) [reply README.md]: Okay I'll send in a PR to remove it from the Symfony docs 👍

**Pull request closed without merging**